frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx

```diff
@@ -100,8 +102,18 @@ function MobileView(props: CodingBountiesProps) {
   const [keysendStatus, setKeysendStatus] = useState(false);
   const [lnInvoice, setLnInvoice] = useState('');
   const [toasts, setToasts]: any = useState([]);
+  const [updatingPayment, setUpdatingPayment] = useState<boolean>(false);
+
+  let bountyPaid = paid || invoiceStatus || keysendStatus;
+
+  if (localPaid === 'PAID') {
+    bountyPaid = true;
+  }
+
+  if (localPaid === 'UNPAID') {
```

Comment:
why not do else if here?

Reply:
Hmmmm, that's true. I can't remember why I did it that way.

But will push an update with my next PR. Thanks